docs/source/modules/introduction.rst

```diff
@@ -0,0 +1,111 @@
+Introduction to running your own Isca experiment.
+==============
+
+Summary
+-------
+Once you have managed to run a test case, you may want to run your own experiment.
```

Comment:
Maybe add a sentence after this one along the lines: `The model configuration, diagnostics, and simulation parameters of your experiment can be configured with python-based run script.`